Clean up `Song content rules` and `Explicit content` (#6048)
ppy/osu-wiki: wiki/Rules/Explicit_Content/en.md

![Screenshot of a beatmap that contains explicit content](img/explicit-tag.jpg "An example of a beatmap that is being marked with the 'explicit' tag.")

Mappers are able to declare whether the content of their beatmap is *explicit* by using a check box in the [genre and language popup](/wiki/Beatmap/Genre_and_language) on the website. This feature has a few expectations and rules governing its use.

Regardless of the nature of audio, make sure it also complies with the [general song content rules](/wiki/Rules/Song_Content_Rules).

## What counts as explicit content?

> Explicit content refers primarily to the **audio content of a beatmap**, mostly in theme, subject matter or *heavy* use of explicit language. It does **not** apply to visual elements of a beatmap — those **must** adhere to the [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations).**

Reviewer: ```suggestion
Explicit content refers primarily to the **audio content of a beatmap**, mostly in theme, subject matter or *heavy* use of explicit language. It does **not** apply to visual elements of a beatmap—those **must** adhere to the [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations).**
```

> In certain circumstances, some exclusions may apply for artistic use of high-effort content, such as [storyboards](/wiki/Storyboards) or bespoke (ie: specifically created) video content.

Reviewer: ```suggestion
In certain circumstances, some exclusions may apply for artistic use of high-effort content, such as [storyboards](/wiki/Storyboards) or bespoke (i.e. specifically created) video content.
```

Generally speaking, with the advent of the new explicit content marker, most forms of music are now acceptable to use in beatmaps with very few exceptions, so long as they are marked appropriately.

## What counts as heavy use of explicit language?

> Mild (or single) use of any common profanity is not enough to qualify a beatmap for the explicit marker. Beatmaps with this kind of language can be safely left without being tagged as explicit — so long as the language is what one would reasonably expect from something classified as requiring "parental guidance" (PG-13). If any arguments break out over whether this is relevant for a given track, assume that it isn't.

Reviewer: ```suggestion
Mild (or single) use of any common profanity is not enough to qualify a beatmap for the explicit content marker. Beatmaps with this kind of language can safely be left without being tagged as explicit—so long as the language is what one would reasonably expect from something classified as requiring "parental guidance" (PG-13). If any arguments break out over whether this is relevant for a given track, assume that it is not.
```

> Profanity use must be sustained, significant, heavily inflammatory or highly repetitive in order to count as explicit.

Reviewer: ```suggestion
Profanity use must be sustained, significant, heavily inflammatory, or highly repetitive in order to count as explicit.
```

![Screenshot of a beatmap that contains explicit content](img/explicit-tag.jpg "An example of a beatmap that is being marked with the 'explicit' tag.")

Mappers are able to declare whether the content of their beatmap is *explicit* by using a check box in the [genre and language popup](/wiki/Beatmap/Genre_and_language) on the website. This feature has a few expectations and rules governing its use.

Regardless of the nature of the audio, make sure it also complies with the [general song content rules](/wiki/Rules/Song_Content_Rules).

## What counts as explicit content?

Explicit content refers primarily to the **audio content of a beatmap**, mostly in theme, subject matter, or *heavy* use of explicit language. It does **not** apply to visual elements of a beatmap—those **must** adhere to the [Visual Content Considerations](/wiki/Rules/Visual_Content_Considerations).**

> In certain circumstances, some exclusions may apply for artistic use of high-effort content, such as [storyboards](/wiki/Storyboards) or bespoke (i.e. specifically created) video content.

Reviewer 2: I find this reads really really weird. I'm guessing it's talking about visual content now? If so it should be on the linked page. If not, there needs to be some restructuring/rewording, because I don't know how "storyboard" is related to audio.

Author: > how "storyboard" is related to audio

it isn't; this paragraph is here since the moment the article was written (https://github.com/ppy/osu-wiki/commit/bdeb2944eb4f44292951adbbe2659d17cc9196ed), and *I guess* it is inserted for a smooth transition to the visual content consideration article (and exceptions are well-documented in there)

Author: I've removed that part, should read better now
